accepted/00000-maintenance-windows.md

```diff
@@ -0,0 +1,191 @@
+- Feature Name: Maintenance Windows
+- Start Date: 2020-03-31
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Define Maintenance Windows in Uyuni and assign them to systems
+
+# Motivation
+[motivation]: #motivation
+
+Customers have company policies to perform changes on systems only at specific points in time.
+This is typically called "Maintenance Window".
+
+Uyuni should be able to define them and prevent executing of actions outside of such a window.
+
+# Architecture
+[architecture]: #architecture
+
+A system is in "Maintenance Mode" or not.
+When a system is in "Maintenance Mode" no action can be executed.
```

Comment:
totally right. I will change it

Reply:
I was also thinking like that subconsciously: having a system in _that_ mode means the system is "locked". Maybe it's because I'm used to the "lock" and "blackout" mode that this similar semantics.

I _really_ like the maintenance mode/window/schedule terminology and can't think of anything better with the "inverse" semantics (the best I could come up with, was "production" or "protected", but I don't like this, or "locked", which we already use), but maybe somebody else has a better idea. For me, the "maintenance" is still the best option.